### PR TITLE
fix listing of drives and quota

### PR DESCRIPTION
**Pull request by micbar**
## Description

Fix listing of drives on a single user.

## Related Issue

- Fixes <issue_link>

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Review by kobergj — APPROVED** [review]

**Review by JanAckermann — APPROVED** [review]

File: changelog/unreleased/add-drives-endpoint-and-userfilter.md [hunk 1]
```diff
@@ -4,4 +4,5 @@ We have added `$expand=drives` to the `/users/{id}/` endpoint
 using the user filter implemented in reva.
 
 https://github.com/owncloud/ocis/pull/4072
-https://github.com/cs3org/reva/pull/3046
\ No newline at end of file
+https://github.com/cs3org/reva/pull/3046
+https://github.com/cs3org/reva/pull/4323
```
Comment (ScharfViktor) [inline changelog/unreleased/add-drives-endpoint-and-userfilter.md:8]: inexistent link

File: services/graph/pkg/service/v0/users.go [hunk 2]
```diff
@@ -180,8 +181,11 @@ func (g Graph) GetUser(w http.ResponseWriter, r *http.Request) {
 	if slices.Contains(sel, "drive") || slices.Contains(sel, "drives") || slices.Contains(exp, "drive") || slices.Contains(exp, "drives") {
 		wdu, err := url.Parse(g.config.Spaces.WebDavBase + g.config.Spaces.WebDavPath)
 		f := listStorageSpacesUserFilter(user.GetId())
+		// use the unrestricted flag to get all possible spaces
+		// users with the canListAllSpaces permission should see all spaces
```
Comment (ScharfViktor) [inline services/graph/pkg/service/v0/users.go:185]: Will an admin have this permissions by default? I need it for tests

File: changelog/unreleased/add-drives-endpoint-and-userfilter.md (hunk 1 above)
Comment (micbar) [reply changelog/unreleased/add-drives-endpoint-and-userfilter.md:7]: fixed, thanks

File: services/graph/pkg/service/v0/users.go (hunk 2 above)
Comment (micbar) [reply services/graph/pkg/service/v0/users.go:185]: The permission `listAllSpaces` is attached to the SpaceAdmin Role.

File: changelog/unreleased/add-drives-endpoint-and-userfilter.md (hunk 1 above)
Comment (ScharfViktor) [reply changelog/unreleased/add-drives-endpoint-and-userfilter.md:7]: probably you deleted the correct link (3046). Wrong link(4323) left

Comment (micbar) [reply changelog/unreleased/add-drives-endpoint-and-userfilter.md]: fixed again 😄

**Review by dragonchaser — APPROVED** [review]

**Review by ScharfViktor — APPROVED** [review]

**Pull request merged**